### ISSUE TITLE
Empty string output

### ISSUE THREAD

**Issue by dontwork** [yakovmeister/pdf2image#84]
```
const options = {
    density: 300, // ppi
    format: "png",
    width,
    height
  }
  return pdf2pic.fromBuffer(buffer, options).bulk(1, true)
    .then(resizedImageBase64 => {
```

Hi I use your library regularly, thank you for making it.

I recently had an issue where the `resizedImageBase64` above was an empty string. I was actually unable to replicate the issue on either my windows machine or within WSL.... I have it running in docker with the imagemagick and ghostscript packages installed and this is where the error is occurring.

However some research showed that ghostscript might silently fail if a font is not included in the PDF and the PDF i had the issue did not have the font included that the text used. I was hoping it might be possible to pass options through to ghostscript through this library in order to test this hypothesis. Is this possible?

Thanks

**Comment by kuzdogan** [comment]
I also have the same issue. Here's my code
```js
  let res = await fetch(url);
  let buffer = await res.buffer();

  const options = {
    width: 2550,
    height: 3300,
    density: 330,
    savePath: "./output/from-buffer-to-base64"
  }

  // let convert = fromPath('pdf.pdf', options);
  let convert = fromBuffer(buffer, options);
  console.log(convert)

  let output = await convert(1, true)
  console.log('output')
  console.log(output)

  let base64 = output.base64;
  console.log('base64')
  console.log(base64)
```

`fetch` is the npm library `node-fetch` here.

**Comment by kuzdogan** [comment]
Ok this is dumb but I actually didn't install the dependencies `ghostscript` and `graphicsmagick` :facepalm:. Now it works. Just install it as [here](https://github.com/yakovmeister/pdf2image/blob/next/docs/gm-installation.md).

Unfortunately I won't be able to use the library since I'll use it in a Google Cloud Function.

**Comment by simaybocu** [comment]
![2022-04-19 (6)](https://user-images.githubusercontent.com/33957028/163978013-8b421444-766e-4190-b0b3-5dd101a08923.png)
When I try to install the ghostscript4js package it gives this error. What could it be from? Can you please help?

**Comment by simaybocu** [comment]
> Ok this is dumb but I actually didn't install the dependencies `ghostscript` and `graphicsmagick` 🤦. Now it works. Just install it as [here](https://github.com/yakovmeister/pdf2image/blob/next/docs/gm-installation.md).
> 
> Unfortunately I won't be able to use the library since I'll use it in a Google Cloud Function. If you are looking for an alternative like me you can use external API's such as [cloudconver](cloudconvert.com)
![2022-04-19 (6)](https://user-images.githubusercontent.com/33957028/163978229-a63810ee-a078-4c55-9d2d-752027abc4ad.png)
When I try to install the ghostscript4js package it gives this error. What could it be from? Can you please help?

**Comment by kuzdogan** [comment]
> > Ok this is dumb but I actually didn't install the dependencies `ghostscript` and `graphicsmagick` facepalm. Now it works. Just install it as [here](https://github.com/yakovmeister/pdf2image/blob/next/docs/gm-installation.md).
> > Unfortunately I won't be able to use the library since I'll use it in a Google Cloud Function. If you are looking for an alternative like me you can use external API's such as [cloudconver](cloudconvert.com)
> > ![2022-04-19 (6)](https://user-images.githubusercontent.com/33957028/163978229-a63810ee-a078-4c55-9d2d-752027abc4ad.png)
> > When I try to install the ghostscript4js package it gives this error. What could it be from? Can you please help?

I think I was referring to the Ubuntu packages: https://github.com/yakovmeister/pdf2image/blob/master/docs/gm-installation.md Follow the Windows instructions there

**Comment by simaybocu** [comment]
> > > Ok this is dumb but I actually didn't install the dependencies `ghostscript` and `graphicsmagick` facepalm. Now it works. Just install it as [here](https://github.com/yakovmeister/pdf2image/blob/next/docs/gm-installation.md).
> > > Unfortunately I won't be able to use the library since I'll use it in a Google Cloud Function. If you are looking for an alternative like me you can use external API's such as [cloudconver](cloudconvert.com)
> > > ![2022-04-19 (6)](https://user-images.githubusercontent.com/33957028/163978229-a63810ee-a078-4c55-9d2d-752027abc4ad.png)
> > > When I try to install the ghostscript4js package it gives this error. What could it be from? Can you please help?
> 
> I think I was referring to the Ubuntu packages: https://github.com/yakovmeister/pdf2image/blob/master/docs/gm-installation.md Follow the Windows instructions there

yes, I installed both under C:\Program Files, but it still doesn't work, base64 is an empty string :(( Do you know a different package with a similar function?

**Comment by AndreasSauer** [comment]
Ich bekomme leider auch nur einen leeren String zurück. Mache ich irgendwas falsch. Ich habe ghostscript und graphicsmagick installiert. Gibt es sonst noch etwas beachten? Aktuell benutze ich Windows 10.

**Comment by AndreasSauer** [comment]
Unfortunately, I also only get an empty string back. Am I doing something wrong? I have installed ghostscript and graphicsmagick. Is there anything else I should be aware of? I am currently using Windows 10.
![screenshot_cmd](https://user-images.githubusercontent.com/17124958/172919234-ca21f4e8-2b45-404e-8cea-2a8a45276174.png)

**Comment by Suraj59** [comment]
is there any update regarding the issue? guys, it's working fine before now it's broken.
